manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ROOT: analyzing petabytes of data, scientifically.",
+  "short_name": "ROOT",
+  "description": "ROOT Website",
+  "theme_color": "#346295",
+  "background_color": "#346295",
+  "display": "standalone",
+  "orientation": "portrait",
```

Comment:
I think landscape would be equally fine, no? Can we just leave this out, to allow both orientations?

Reply:
Yes, we can do that. I just saw it on the guide I was using and assumed it was mandatory 😅